### PR TITLE
Update de.yml

### PR DESCRIPTION
**Pull request by RoyalCrafter**
Translated everything to german

### CODE REVIEW

File: locales/de.yml [hunk 1]
```diff
@@ -1,76 +1,76 @@
...
     menu_title:
-      new: New Chat
-      history: Chat History
-      prompts: Prompts
-      export_md: Export Markdown
-      settings: Settings
-      hide_side: Hide Side
+      new: Neuer Chat
+      history: Chatverlauf
+      prompts: Eingabeaufforderung
```
Comment (secondtruth) [inline locales/de.yml:50]: `Aufforderungen`?

File: locales/de.yml [hunk 2]
```diff
@@ -1,76 +1,76 @@
...
 window:
   message:
-    close: Do you want to exit the application when you click the [x] button?
+    close: Willst du die Anwendung schließen, wenn du [x] drückst?
 menu:
   app:
-    about: About NoFWL
-    check_update: Check for Updates
+    about: Über NoFWL
+    check_update: Auf Updates überprüfen
```
Comment (secondtruth) [inline locales/de.yml:9]: `Auf Updates prüfen`?

File: locales/de.yml [hunk 3]
```diff
@@ -1,76 +1,76 @@
...
   preferences:
-    name: Preferences
-    control_center: Control Center
-    stay_on_top: Stay On Top
-    titlebar: Titlebar
-    hide_dock_icon: Hide Dock Icon
-    language: Language
+    name: Einstellungen
+    control_center: Kontrollzentrum
+    stay_on_top: Immer oben bleiben
```
Comment (secondtruth) [inline locales/de.yml:13]: `Im Vordergrund bleiben`?

File: locales/de.yml [hunk 4]
```diff
@@ -1,76 +1,76 @@
...
+      settings: Einstellungen
+      hide_side: Seite verstecken
     message:
-      chat_history: Chat History
-      chat_delete: Are you sure you want to delete this chat record? It cannot be restored after deletion!
+      chat_history: Chatverlauf
+      chat_delete: Bist du dir sicher, dass du diesen Chat löschen willst? Er kann nicht wiederhergestellt werden!
   modal:
-    settings_title: GPT Settings
+    settings_title: GPT Einstellungen
```
Comment (secondtruth) [inline locales/de.yml:58]: `GPT-Einstellungen`

File: locales/de.yml [hunk 5]
```diff
@@ -1,76 +1,76 @@
...
   help:
-    name: Help
-    app_log: NoFWL Log
-    report_bug: Report Bug
+    name: Hilfe
+    app_log: NoFWL-Protokoll
+    report_bug: Bug melden
     feedback: Feedback
-    dev_tools: Toggle Developer Tools
+    dev_tools: Entwicklerwerkzeuge anzeigen/verstecken
```
Comment (secondtruth) [inline locales/de.yml:40]: `Entwicklerwerkzeuge umschalten`?

**Review by MrDrache333 — APPROVED** [review]

**Review by secondtruth — COMMENTED** [review]
I added some translation suggestions.

File: locales/de.yml (hunk 5 above)
Comment (RoyalCrafter) [reply locales/de.yml:40]: I thought about that, but it sounds a bit strange in my opinion

File: locales/de.yml (hunk 2 above)
Comment (RoyalCrafter) [reply locales/de.yml:9]: I thinks it's personal preference
`Nach Updates suchen`would also be correct

**Pull request merged**